app/controllers/api/v1/books_controller.rb

```diff
@@ -0,0 +1,21 @@
+module Api
+  module V1
+    class BooksController < ApplicationController
+      before_action :authenticate_api_v1_user!
+
+      def index
+        @books = Book.all
+        render_paginated @books, each_serializer: BookIndexSerializer
+      end
+
+      def show
+        @book = Book.find_by(id: params[:id])
+        if @book
```

Comment:
<img width="857" alt="screen shot 2018-10-24 at 1 16 20 pm" src="https://user-images.githubusercontent.com/43754771/47452120-0a6cd580-d78f-11e8-87e9-7942d4791345.png">

rubocop shows me and error with this way, if i use like you say, appears the first issue, and if I add a newline rubocop says that there is a Trailing whitespace detected... so... What could I do?

Reply:
the first error it's ok, you must add a newline,
the second one it's because you are leaving spaces in that new line the identation for that line should be 0